samples/apps/copilot-chat-app/webapi/SemanticKernelExtensions.cs

```diff
@@ -156,6 +162,17 @@ private static void AddSemanticTextMemory(this IServiceCollection services)
         services.AddSingleton<OptionalIMemoryStore>(sp => new OptionalIMemoryStore() { MemoryStore = sp.GetService<IMemoryStore>() });
     }
 
+    /// <summary>
+    /// Adds Azure Content Moderator
+    /// </summary>
+    internal static void AddContentModerator(this IServiceCollection services)
+    {
+        IConfiguration configuration = services.BuildServiceProvider().GetRequiredService<IConfiguration>();
+        ContentModerationOptions options = configuration.GetSection(ContentModerationOptions.PropertyName).Get<ContentModerationOptions>();
```

Comment:
Why not simply

ContentModeratiomOptions config = services.BuildServiceProvider().GetRequiredService<IOptions<ContentModerationOptions>>().Value;

Reply:
I tried that logic but it wasn't loading in the secrets set using dotnet set-secrets correctly? Not sure if there's a race condition there so had to load the configuration section explicitly

Reply:
Strange, oh well... I won't any sleep over this